apps/prairielearn/src/pages/studentDetail/studentDetail.html.tsx

```diff
@@ -0,0 +1,187 @@
+        ) : (
+          <table class="table table-sm table-hover" aria-label="Student Assessment Performance">
+            <thead>
+              <tr>
+                <th style="width: 1%">
+                  <span class="visually-hidden">Label</span>
+                </th>
+                <th>
+                  <span class="visually-hidden">Assessment</span>
+                </th>
+                <th class="text-center">Score</th>
+                <th class="text-center">Points</th>
+                <th class="text-center">Actions</th>
+              </tr>
+            </thead>
+            <tbody>
+              {Array.from(gradebookRowsBySet.entries()).map(([setHeading, setAssessments]) => (
+                <Fragment key={setHeading}>
+                  <tr>
+                    <th colspan={5}>{setHeading}</th>
+                  </tr>
+                  {setAssessments.map((row) => (
+                    <tr key={row.assessment.id}>
+                      <td class="align-middle" style="width: 1%">
+                        <span class={`badge color-${row.assessment_set.color}`}>
+                          {computeLabel(row)}
+                        </span>
+                      </td>
+                      <td class="align-middle">
+                        {computeTitle(row)}
+                        {row.assessment.group_work && (
+                          <i class="fas fa-users ms-1" aria-hidden="true" title="Group work" />
+                        )}
+                      </td>
+                      <td class="text-center align-middle">
+                        {row.assessment_instance.id && row.show_closed_assessment_score ? (
+                          <Scorebar score={row.assessment_instance.score_perc} class="mx-auto" />
+                        ) : row.assessment_instance.id ? (
+                          'In progress'
+                        ) : (
+                          <span class="text-muted">Not started</span>
+                        )}
+                      </td>
+                      <td class="text-center align-middle">
+                        {row.assessment_instance.id && row.show_closed_assessment_score ? (
+                          `${row.assessment_instance.points?.toFixed(1) || '0.0'} / ${row.assessment_instance.max_points?.toFixed(1) || '0.0'}`
+                        ) : row.assessment_instance.id ? (
+                          <span class="text-muted">—</span>
+                        ) : (
+                          <span class="text-muted">—</span>
+                        )}
+                      </td>
+                      <td class="text-center align-middle">
+                        {row.assessment_instance.id ? (
+                          <a
+                            href={`${urlPrefix}/assessment_instance/${row.assessment_instance.id}`}
+                            class="btn btn-xs btn-outline-primary"
+                          >
+                            View Instance
```

Comment:
Suggesting `Sentence casing`.
```suggestion
                            View instance
```